src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,44 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+import { findTab } from '../../utils';
+
+export const Tabs = ({ tabs, onTabSelected, selectedTabId }) => {
+  const currentTab = findTab(tabs, selectedTabId) || tabs[0];
```

Comment:
In the `findTab` function, you already return the first tab if nothing is found, so you don't need to do it again.